packages/eslint-plugin/docs/rules/no-deprecated.mdx

```diff
@@ -59,6 +59,59 @@ const url2 = new URL('/foo', 'http://www.example.com');
 </TabItem>
 </Tabs>
 
+## Options
+
+### `allow`
+
+{/* insert option description */}
+
+This option takes the shared [`TypeOrValueSpecifier` format](/packages/type-utils/type-or-value-specifier).
+
+Examples of code for this rule with:
+
+```json
+{
+  "allow": [
+    { "from": "file", "name": "apiV1" },
+    { "from": "lib", "name": "HTMLFrameElement" },
+    { "from": "package", "name": "Bar", "package": "bar-lib" }
+  ]
+}
+```
+
+<Tabs>
+<TabItem value="❌ Incorrect">
+
+```ts option='{"allow":[{"from":"file","name":"apiV1"},{"from":"lib","name":"HTMLFrameElement"},{"from":"package","name":"Bar","package":"bar-lib"}]}'
+/** @deprecated */
+declare function apiV2(): Promise<string>;
+
+await apiV2();
+
+function foo(
+  element: HTMLFrameSetElement, // HTMLFrameSetElement is deprecated (not specified in the option)
```

Comment:
`HTMLFrameSetElement` is deprecated, so when I check it in the playground, it shows a lint error. However, snapshot tests don't show lint errors, so maybe this is a bug in the snapshot test tsconfig configuration?